product-features/liquidity-pool/self-service-listing.md

```diff
@@ -1,122 +1,101 @@
 ---
-coverY: 0
-layout:
-  cover:
-    visible: false
-    size: full
-  title:
-    visible: true
-  description:
-    visible: false
-  tableOfContents:
-    visible: true
-  outline:
-    visible: true
-  pagination:
-    visible: true
+description: Create your own pool and make your token tradeable on ALEX's decentralized exchange in simple steps!
 ---
 
-# 😀 Self Service Listing
+# 📝 Self-Service Listing
 
-## Self-Service Listing Procedure
+## Getting Started
 
-Step 1: Submit the token information & deposit the anchor token (Token Y)
+### What is self-service listing?
 
-Step 2: Waiting for the pool contract creation (around 24-48 hours)
+Self-service listing allows you to list your own token on ALEX's decentralized exchange, enabling the permissionless trade of your token against a so-called anchor token within the exchange.
 
-Step 3: Deposit the listing token (Token X)
+ALEX decentralized exchange (DEX) is an Automated Market Maker (AMM) type of exchange. This means it operates with trading pools. These pools consist of a pair of tokens with sufficient funds (liquidity) of each to ensure a fair trade between the tokens that make up the pool.
 
-Step 4: Open Pool
+In this context, listing a token involves launching a new liquidity pool containing the new token and the anchor token on ALEX's AMM DEX, thereby facilitating trading within the community and enabling price discovery. 
```

Comment:
describe what is an anchor token.

Reply:
@XianPaz fijate que ahí hice push addressando esto, decime qué te parece. Sobre todo si está bien dónde lo puse (primer párrafo)